iphone/Classes/TiUIWebView.m

```diff
@@ -385,6 +386,38 @@ -(BOOL)canGoForward
 	return [webview canGoForward];
 }
 
+-(void)setBlacklistedURLs_:(id)value
+{
+    if (blacklistedURLs == nil) {
+        blacklistedURLs = [[[NSMutableArray alloc] init] retain];
+    }
```

Comment:
We need to release and re-create the array every-time we call this function (as discussed), so make sure it gets releases properly and set it again.